tracepoint_perf/examples/decode_perf.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+//! Demonstrates how to use [`tp::PerfDataFileReader`] to decode events from a
+//! `perf.data` file.
+
+use std::env;
+use std::process;
+use std::vec;
+
+use tracepoint_decode as td;
+use tracepoint_perf as tp;
+
+fn main() -> process::ExitCode {
```

Comment:
Note: This produces output that is almost identical to the output from LinuxTracepoints-Net DecodeSample.